Makefile

```diff
@@ -130,3 +156,16 @@ CONTROLLER_GEN=$(GOBIN)/controller-gen
 else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
+
+code-generator:
+ifeq (, $(shell which code-generator))
+	@{ \
+	cd $(GOBIN) ;\
```

Comment:
My `GOBIN` is empty...
```
go env
...
GOBIN=""
```

Let's put the zipfile in a temp-dir like CONTROLLER-GEN does.
```
	set -e ;\
	CODE_GEN_TMP_DIR=$$(mktemp -d) ;\
	cd $$CODE_GEN_TMP_DIR ;\
        curl -L -o code-generator.zip https://github.com/kubernetes/code-generator/archive/refs/tags/v0.21.5.zip ;\
	unzip code-generator.zip ;\
        mv code-generator $(GOPATH)/bin/
```

Reply:
dcf19b9f2985404bafe65bcd4e7d3b20a0291713